### PR TITLE
Enhancement: log company sym when its ability is used for tokens/tiles

### PR DESCRIPTION
**Pull request by michaeljb**
e.g., "ELA (DC&H) places a token" instead of "ELA places a token" to indicate
that ELA used the ability from the company Dewey, Cheatham, and Howe to place
that token

### CODE REVIEW

File: lib/engine/step/tokener.rb [hunk 1]
```diff
@@ -49,9 +55,9 @@ def place_token(entity, city, token, teleport: false, special_ability: nil)
         when :neutral
           entity.tokens.delete(token)
           token.corporation.tokens << token
-          @log << "#{entity.name} places a neutral token on #{hex.name}#{price_log}"
+          @log << "#{tokener} places a neutral token on #{hex.name}#{price_log}"
```
Comment (tobymao) [inline lib/engine/step/tokener.rb:58]: won't tokener be nil?

Comment (michaeljb) [reply lib/engine/step/tokener.rb:58]: tokener gets set in either branch of the if statement at line 40. I could set it to “entity.name” before that block if that would be more clear.

Comment (tobymao) [reply lib/engine/step/tokener.rb:58]: it doesn't every time

 tokener = "#{entity.name} (#{ability.owner.sym})" if ability.owner != entity

File: lib/engine/step/tokener.rb [hunk 2]
```diff
@@ -37,7 +37,13 @@ def place_token(entity, city, token, teleport: false, special_ability: nil)
         @game.game_error('Token is already used') if token.used
 
         token, ability = adjust_token_price_ability!(entity, token, hex, city)
-        entity.remove_ability(ability) if ability
+        if ability
```
Comment (ryandriskel) [inline lib/engine/step/tokener.rb:40]: How about instead:

```
tokener = entity.name
if ability
  tokener = "#{entity.name} (#{ability.owner.sym})" if ability.owner != entity
  entity.remove_ability(ability)
end
```

File: lib/engine/step/tokener.rb (hunk 1 above)
Comment (ryandriskel) [reply lib/engine/step/tokener.rb:58]: How about instead:
```
tokener = entity.name
if ability
  tokener = "#{entity.name} (#{ability.owner.sym})" if ability.owner != entity
  entity.remove_ability(ability)
end
```

Comment (michaeljb) [reply lib/engine/step/tokener.rb:58]: Oh, right of course. Thanks!

**Pull request merged**